sdk/cosmos/azure-cosmos/azure/cosmos/__init__.py

```diff
@@ -26,7 +26,7 @@
 from .database import DatabaseProxy
 from .user import UserProxy
 from .scripts import ScriptsProxy
-from .offer import Offer
```

Comment:
were we not going to expose both of them?

Reply:
from my understanding i thought it was being replaced with throughput properties

Reply:
I can add it back if necessary though